### PR TITLE
Hit the API

### PR DESCRIPTION
**Pull request by Maha-Magdy**
Hello 😊
In this pull request I have made the following:
- created a new game with the name of my choice, which is "My cool new game ^_^" by using the API, and make the query using Postman.
The response to my request is:
```
{
    "result": "Game with ID: 7KZzvMgheMijgkb7Br8c added."
}
```
- Implemented the "Refresh" button (receiving data from the API and parsing the JSON).
- Implemented the form "Submit" button (sending data to the API).
- Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,23 +1,43 @@
...
-      document.createTextNode(
-        `${recentScores[i].name}: ${recentScores[i].score}`,
-      ),
-    );
-    ul.appendChild(li);
-  }
+const recentScoresList = document.getElementById('recent-scores-list');
 
-  recentScoresList.appendChild(ul);
+function getRecentScores() {
```
Comment (rloterh) [inline src/index.js]: - [x] Your function does not conform to the _arrow functions_ syntax. Kindly change the _normal_ function to an _arrow function_ syntax, so as to keep your _function_ syntax simpler and cleaner

File: src/new-record.js [hunk 2]
```diff
@@ -1,6 +1,25 @@
+/* eslint-disable consistent-return */
 export default class NewRecord {
   constructor(name, score) {
     this.name = name;
     this.score = score;
```
Comment (rloterh) [inline src/new-record.js:5]: - [ ] OPTIONAL
Your score input is not validated, and therefore any it accepts any input. Kindly endeavor to improve upon this aspect to improve the security feature of your form

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Maha, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Data is stored to and can be retrieved from API
- Promise feature is well implemented
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comment(s) marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/new-record.js (hunk 2 above)
Comment (Maha-Magdy) [reply src/new-record.js:5]: Thank you for the suggestion, I see it's a very valid point, but I prefer to do it in the final retouch step. Thanks for highlight this point for me.

Comment (henatan99) [reply src/new-record.js:5]: This is just a stretch requirement. It is up to the student to validate input on this milestone. :+1:

File: src/index.js (hunk 1 above)
Comment (henatan99) [reply src/index.js]: This issue is well addressed. Thank you for making the requested changes. :clap:

**Review by henatan99 — APPROVED** [review]
Hi @Maha-Magdy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**